src/signals/my-incidents/context/context.tsx

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import { createContext, useContext } from 'react'
+
+import type { MyIncidentsValue } from '../types'
+
+export const MyIncidentsContext = createContext<MyIncidentsValue | null>(null)
+
+export const useMyIncidentContext = (): MyIncidentsValue => {
+  const context = useContext(MyIncidentsContext)
+
+  if (context === null) {
```

Comment:
why checking it, it seems it will always be there due to the return type of useContext.

Reply:
That is not true. If you forget to wrap your component with the correct provider you will get this error message in the console.